code/modules/martial_arts/combos/superhuman/brutal_barrage.dm

```diff
@@ -0,0 +1,17 @@
+/datum/martial_combo/superhuman/brutal_barrage
+	name = "Brutal Barrage"
+	explaination_text = "We unleash a flurry of swift blows that completely decimates an oppenent in a critical state."
+	steps = list(MARTIAL_COMBO_STEP_HARM, MARTIAL_COMBO_STEP_DISARM, MARTIAL_COMBO_STEP_GRAB, MARTIAL_COMBO_STEP_HARM)
```

Comment:
One more small thing. Having grab and then something afterwards will give a wrong help text. Since using grab and then harm on somebody will do nothing. Since you now have a grab as your hand. Bit annoying thing and people will get confused why it is not working while they follow the instructions to the point